Python/pystate.c

```diff
@@ -2190,7 +2190,7 @@ start_the_world(struct _stoptheworld_state *stw)
     PyThreadState *t;
     _Py_FOR_EACH_THREAD(stw, i, t) {
         if (t != stw->requester) {
-            assert(t->state == _Py_THREAD_SUSPENDED);
+            assert(_Py_atomic_load_int(&t->state) == _Py_THREAD_SUSPENDED);
```

Comment:
Maybe a `_Py_atomic_load_int_relaxed`?

I think we'll generally want to use relaxed atomics in these sorts of assertions to avoid introducing an ordering from the atomic load that only exists in the debug build with assertions enabled.